src/app/page.tsx

```diff
@@ -85,7 +85,7 @@ const HomePage = async () => {
               </FadeInView>
             </div>
           </div>
-          <div className="col-span-1">
+          <div className="col-span-1 p-4 sm:p-0">
```

Comment:
Center the contents of this column

```suggestion
          <div className="col-span-1 p-4 sm:p-0 flex items-center justify-center">
```

<sub>_[Suggestion by Codeium Forge](https://forge.codeium.com/khou22/khou22.github.io/37/review)_</sub>